src/Dev/TestSession.php

```diff
@@ -313,24 +323,13 @@ public function cssParser()
     }
 
     /**
-     * Get the last response as a SimplePage object
+     * Get a DOM Crawler for the last response
      *
-     * @return SimplePage The response if available
+     * A DOM Crawler for the given response
```

Comment:
```suggestion
```
This doesn't actually add any information that the return type itself and the comment above don't already provide.

Also for future reference a return comment in phpdoc must be preceded by `@return` and the type.

Reply:
DONE